clang/test/3C/arrboundsprototypemerge.c

```diff
@@ -0,0 +1,24 @@
+// RUN: rm -rf %t*
+// RUN: 3c -base-dir=%S -alltypes %s -- | FileCheck -match-full-lines %s
+// RUN: 3c -base-dir=%S -alltypes %s -- | %clang -c -f3c-tool -fcheckedc-extension -x c -o %t1.unused -
+
+/*
+Handle function prototype merging in bounds inference.
+https://github.com/correctcomputation/checkedc-clang/issues/522
+*/
+
+#include <string_checked.h>
```

Comment:
We could just use `<string.h>` now that we have implicit headers. However, we generally don't like to include checked headers in tests. Can we inline the prototypes here in this file in order to test them that way, like we often do for `malloc`, `free`, etc.?